.pre-commit-config.yaml

```diff
@@ -20,6 +20,7 @@ repos:
     rev: v0.3.4
     hooks:
       - id: blackdoc
+        exclude: "generate_reductions.py"
```

Comment:
@max-sixty, FYI it is possible to specify the `force_exclude` and `exclude` options (and also all others, but I didn't verify that) in the `tool.blackdoc` section. You'd probably have to use `force_exclude` because `pre-commit` specifies the changed files explicitly.